dspace-api/src/main/java/org/dspace/core/Utils.java

```diff
@@ -411,7 +425,33 @@ public static synchronized String formatISO8601Date(Date d)
         return result.substring(0, rl-2) + ":" + result.substring(rl-2);
     }
 
-    public static <E> Collection<E> emptyIfNull(Collection<E> collection) {
+    public static <E> java.util.Collection<E> emptyIfNull(java.util.Collection<E> collection) {
         return collection == null ? Collections.<E>emptyList() : collection;
     }
+    
+	public static String[] tokenize(String metadata) {
+		String separator = metadata.contains("_") ? "_" : ".";
+		StringTokenizer dcf = new StringTokenizer(metadata, separator);
+
+		String[] tokens = { "", "", "" };
+		int i = 0;
+		while (dcf.hasMoreTokens()) {
+			tokens[i] = dcf.nextToken().trim();
+			i++;
+		}
+		// Tokens contains:
+		// schema = tokens[0];
+		// element = tokens[1];
+		// qualifier = tokens[2];
```

Comment:
These comments should probably be in the Javadocs for this method. That way they are easily found.